crux-core/resources/query-allowlist.edn

```diff
@@ -0,0 +1,79 @@
+#{clojure.core/false? clojure.core/-' clojure.core/ratio?
```

Comment:
doubt we're going to read these often, but could they be one per line and sorted alphabetically?

(emacs will `M-x sort-lines`, remember to temporarily put the `#{` and `}` on their own lines, lest they be sorted too)